packages/react-server-examples/bike-share/pages/index.js

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import {ReactServerAgent, RootContainer, RootElement} from 'react-server';
+import NetworkList from '../components/network-list';
+import Header from '../components/header';
+import Footer from '../components/footer';
+
+export default class IndexPage {
+	handleRoute(next) {
+		this.data = ReactServerAgent.get('http://api.citybik.es/v2/networks');
+		return next();
+	}
+	getElements() {
+		return [
+			<RootContainer key={0}>
```

Comment:
What's this key?

Also, doesn't seem like there's any need for a `<RootContainer>` here...?